src/CoreWCF.Primitives/src/CoreWCF/Security/Tokens/InitiatorServiceModelSecurityTokenRequirement.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace CoreWCF.Security.Tokens
 {
-    internal sealed class InitiatorServiceModelSecurityTokenRequirement : ServiceModelSecurityTokenRequirement
+    public sealed class InitiatorServiceModelSecurityTokenRequirement : ServiceModelSecurityTokenRequirement
```

Comment:
I made this internal as InitiatorServiceModelSecurityTokenRequirement is generally used on client side. Do you know of a server side scenario which needs this?

Reply:
Not yet found.. reverted to internal